include/CoreGen/StoneCutter/Passes/SCLinAlg.h

```diff
@@ -0,0 +1,64 @@
+//
+// _SCAlgorithms_h_
+//
+// Copyright (C) 2017-2022 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+
+/**
+ * \class SCAlgorithms
+ *
+ * \ingroup StoneCutter
+ *
+ * \brief StoneCutter CodeGen Pass: Passes as a Master's Thesis
+ *
+ */
+
+#ifndef _STONECUTTER_SCPASS_SCMETADATAADJUST_H_
```

Comment:
This should be `_STONECUTTER_SCPASS_SCLINALG_H_`; otherwise this header won't get included